dts/arm/st/g4/stm32g484.dtsi

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2021 Guðni Már Gilbert
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/g4/stm32g4.dtsi>
```

Comment:
if this file is empty, `#include <st/g4/stm32g4.dtsi>` may be used instead

Reply:
I forgot to add the DMA nodes similar to the other files. Will fix that quickly.